test/wrappers/ortools_test.rb

```diff
@@ -2527,6 +2527,7 @@ def test_route_force_start
         start_point_id: 'point_0',
         end_point_id: 'point_0',
         matrix_id: 'matrix_0',
+        timewindow: { start: 0 },
```

Comment:
It should not be necessary anymore since we are using grape 1.5.1

Reply:
I added it because I add a checker to say there can not be force_start option activated if vehicle has no timewindow start. Does it make sense for you ?

Reply:
start is not 0 by default? I can't imagine another default value for start. If we had the proper default value for it, then that checker and this modification would be unnecessary, no?

Reply:
We can use 0 as start default value. I guess that would make some parts of the code cleaner.
@braktar do you see any case where that would be uncompatible with current logic ? I do not see how we could be using negative start values anyway.

Reply:
Ok for me to set 0 as default value for the start of a timewindow